src/redux/books/books.js

```diff
@@ -0,0 +1,46 @@
+const ADD_BOOK = 'src/redux/books/ADD_BOOK';
+const REMOVE_BOOK = 'src/redux/books/REMOVE_BOOK';
+const initialState = [];
+/* eslint-disable*/
+const bookReducer = (state = initialState, action) => {
+    /* eslint-enable */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.